java/impl/aws/src/main/java/co/worklytics/psoxy/AWSConfigProperty.java

```diff
@@ -3,6 +3,16 @@
 import co.worklytics.psoxy.gateway.ConfigService;
 
 public enum AWSConfigProperty implements ConfigService.ConfigProperty {
+
+    /**
+     * execution role of this lambda instance
+     *
+     * can we get it from env var? don't see a way:
```

Comment:
Didn't try it, but it seems there is a way according to [this](https://stackoverflow.com/a/62179575) response. If not, we could store the role in a env variable ourselves as any other one

Reply:
thanks - it's actually unneeded finally; i'm pretty confident the 'role' is name of a role in Vault, rather than in AWS. And it's Vault configuration that binds AWS role to Vault role.